### ISSUE TITLE
Group prefix repeating

### ISSUE THREAD

**Issue by Micnubinub** [elysiajs/elysia#86]
Hey, I just updated to `"@elysiajs/eden": "0.6.0"` and `"elysia": "0.6.3"`... Now my group path names are repeating in both in Eden Treaty and Eden Fetch... Any idea how to fix?
<img width="693" alt="SCR-20230810-ibvi-2" src="https://github.com/elysiajs/elysia/assets/4921356/00bed525-ac30-40fe-b31f-c5d9962e5280">

<img width="682" alt="SCR-20230810-iccf-2" src="https://github.com/elysiajs/elysia/assets/4921356/4f2ba799-75c1-4c1f-9a8e-2c2277244132">

**Comment by rhkdgns95** [comment]
I'm having the same problem.

There is an issue in which the group name is repeated when the files of each router are imported and grouped.
Therefore, when executing the request, an error occurs (because there is a problem with the mapping being doubled).

The issue hasn't been resolved, so why is it closed?

Did you also solve the problem?

**Comment by rhkdgns95** [comment]
@gtrabanco oh thanks!

**Comment by SaltyAom** [comment]
@Micnubinub, can you provide a your minimal reproducible code for me to test

**Comment by Micnubinub** [comment]
> @Micnubinub, can you provide a your minimal reproducible code for me to test

 "@elysiajs/eden": "0.6.0",
 "elysia": "0.6.10",

Server:
```ts
const test = (app: Elysia) => app
  .group('/app', group => group
    .get('/test', async () => {
        return "Test"
    }))

const app = new Elysia()
  .use(test)
```

Client:
```ts
export const fetcher = edenFetch<App>(import.meta.env.VITE_BASE_URL, {
})
```
<img width="866" alt="image" src="https://github.com/elysiajs/elysia/assets/4921356/8708be3f-9dd7-4e5a-8981-1fdebe8d758c">

**Comment by Micnubinub** [comment]
@SaltyAom are you able to reproduce?

**Comment by Micnubinub** [comment]
Fixed in "@elysiajs/eden": "0.6.1" and "elysia": "0.6.11"
